src/test/java/api/Address.java

```diff
@@ -0,0 +1,267 @@
+package api;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import jdk.nashorn.internal.objects.annotations.Getter;
+import jdk.nashorn.internal.objects.annotations.Setter;
+
+@JsonDeserialize(as = Address.class)
+public class Address {
```

Comment:
Please locate pojo class in the resources/dto directory (I know we locate pojo here on the lesson)